R/tm_data_table.R

```diff
@@ -162,8 +163,9 @@ ui_page_data_table <- function(id,
   )
 }
 
-
-# server page module
+# server page module.
```

Comment:
You do not have dots at the end of comments in this PR
https://github.com/insightsengineering/teal.modules.general/pull/653/files
Please unify
```suggestion
# server page module
```

Reply:
done. removed `.`